diff --git a/dist/family-chart.js b/dist/family-chart.js
index b54fe3ce78340b0b9f4ed39e2c4a01918acd350b..47bf4b8d745bed01a5319525fe2e277c61e065ae 100644
--- a/dist/family-chart.js
+++ b/dist/family-chart.js
@@ -636,6 +636,8 @@ function CardBody({d,card_dim,card_display}) {
         <text clip-path="url(#card_text_clip)">
           <tspan x="${0}" dy="${14}">${card_display[0](d.data)}</tspan>
           <tspan x="${0}" dy="${14}" font-size="10">${card_display[1](d.data)}</tspan>
+          <tspan x="${0}" dy="${14}" font-size="10">${card_display[2](d.data)}</tspan>
+          <tspan x="${0}" dy="${14}" font-size="10">${card_display[3](d.data)}</tspan>
         </text>
         <rect width="${card_dim.w-card_dim.text_x-10}" height="${card_dim.h-20}" style="mask: url(#fade)" class="text-overflow-mask" /> 
       </g>